pelicanconf.py

```diff
@@ -4,7 +4,7 @@
 
 AUTHOR = u'PyCon India Content Team'
 SITENAME = u'In PyCon Blog'
-SITEURL = '/blog'
+SITEURL = '/'
```

Comment:
You can simply leave this blank

Reply:
Actually we don't need to update this at all. Using SITEURL in the links is the correct fix. The `/blog` url is needed in the devserver while running locally. In Netlify we can use the same `make publish` target. The `RELATIVE_URLS = True` setting overrides the SITEURL setting during publish and uses relative urls everywhere where SITEURL is used.

Reply:
Resolved.